packages/xstate-react/src/fsm.ts

```diff
@@ -36,8 +36,8 @@ export function useMachine<
   }
 ): [
   StateMachine.State<TContext, TEvent, TState>,
-  StateMachine.Service<TContext, TEvent>['send'],
-  StateMachine.Service<TContext, TEvent>
+  StateMachine.Service<TContext, TEvent, TState>['send'],
```

Comment:
I added `TState` to this line too to be consistent with the types I saw elsewhere in the codebase, but I couldn't see any difference with it included or not.